src/app/api/alerts/schemas.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from pydantic import BaseModel, Field, validator
+from app.db import AlertType
+
+
+class AlertIn(BaseModel):
+    device_id: int = Field(..., gt=0)
```

Comment:
Same question as for my previous comment with the "Path". what does the ellipsis do ?

Reply:
It's to specify that the field is required! cf. https://pydantic-docs.helpmanual.io/usage/models/#required-fields